corehq/apps/campaign/static/campaign/js/async_configurable_report.js

```diff
@@ -0,0 +1,428 @@
+                self.widgetFilters.find(self.standardReport.printReportButton).removeClass('d-none');
+            }
+
+            // For ConfigurableReportView, we need to use the correct URL format
+            // The URL should be /a/{domain}/configurable/{report_id}/?format=json
+            var url;
+            
+            // Check if urlRoot is already a complete URL
+            if (self.urlRoot.includes('/configurable/')) {
+                url = self.urlRoot;
+                if (!url.endsWith('/')) {
+                    url += '/';
+                }
+            } else {
+                // Construct the URL for the ConfigurableReportView
+                url = '/a/' + self.domain + '/configurable/' + self.reportId + '/';
+            }
+            
+            // Add query parameters
+            var queryParams = [];
+            if (params) {
+                queryParams.push(params);
+            }
+            if (processFilters) {
+                queryParams.push(processFilters);
+            }
+            queryParams.push('format=json');
+            
+            url += '?' + queryParams.join('&');
+            
+            self.reportRequest = $.ajax({
+                url: url,
+                dataType: 'json',
+                success: function (data) {
+                    self.reportRequest = null;
+                    
+                    // Debug the response
+                    if (data.report) {
+                        self.reportContent.html(data.report);
+                    } else {
+                        self.reportContent.html(
+                            '<div class="alert alert-info">' + 
+                            gettext('No report content in response') + 
+                            '</div>'
+                        );
+                    }
+                    
+                    if (data.filters) {
+                        loadFilters(data);
+                    }
+                    self.issueAttempts = 0;
+                    self.hqLoading = self.reportContent.find(self.loaderClass);
+                    
+                    // Initialize any JavaScript components in the report
+                    try {
+                        // Initialize charts if available
+                        if (typeof chartsMain !== 'undefined' && chartsMain.init) {
+                            chartsMain.init();
+                        }
+                        
+                        // Initialize datatables if available
+                        self.reportContent.find('table.datatable').each(function() {
+                            $(this).DataTable({
+                                responsive: true,
+                                paging: true,
+                                searching: true,
+                                ordering: true,
+                                info: true,
+                                lengthMenu: [[10, 25, 50, 100, -1], [10, 25, 50, 100, "All"]]
```

Comment:
Looking at other HQ code makes me think maybe this should be based on a report param like `defaultRows`, what do you think?

Also, if "All" stays in this list, it should be translated.